### PR TITLE
Add Gem ranking badges

### PR DESCRIPTION
**Pull request by raphink**
This pull request adds badges for Gems ranking, taken from bestgems.org:

* `/gem/rt/<gem>` for total ranking (consolidated)
* `/gem/rd/<gem>` for daily ranking

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3636,6 +3674,11 @@ function fetchFromSvg(request, url, cb) {
   });
 }
 
+function ordinalNumber(n) {
+  var s=["ᵗʰ","ˢᵗ","ⁿᵈ","ʳᵈ"], v=n%100;
```
Comment (espadrine) [inline server.js]: That's pretty cool, but I fear it won't render well:
nth

Comment (raphink) [inline server.js]: Yes... Do you have another suggestion? Normal size letters don't look good
either...

Comment (espadrine) [inline server.js]: I can't think of anything; I fear we are stuck with that choice. If you consider this to look better, I'll merge this.

Comment (raphink) [inline server.js]: I agree it's not perfect, but I still think it's better than normal size text…

**Comment by espadrine** [comment]
After the facts, I think I would consider total ranking the default, and avoid adding `total` in the badge. Do you agree? Can I remove it?

**Comment by raphink** [comment]
Sounds reasonable

**Pull request merged**